cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/FunctionSimulator.java

```diff
@@ -0,0 +1,67 @@
+package ru.urururu.cmakeedit.core.checker;
+
+import ru.urururu.cmakeedit.core.CommandInvocationNode;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by okutane on 22/08/16.
+ */
+public class FunctionSimulator implements AbstractSimulator.CommandSimulator {
+    private final AbstractSimulator simulator;
+    private final List<CommandInvocationNode> body;
+    private Stage stage = Stage.Unused;
+    private SimulationState exitState;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "exitState" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the "exitState" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)